types/wordpress__block-editor/components/media-placeholder.d.ts

```diff
@@ -77,7 +77,7 @@ declare namespace MediaPlaceholder {
          */
         onError?(message: string): void;
         onSelectURL?(src: string): void;
-        multiple?: T | undefined;
+        multiple?: T | string | undefined;
```

Comment:
@mikachan can this be any string or is it limited to specific values? if it's only specific values we should be able to `const` it with a new helper type.

```ts
type MediaPlaceholderMultipleAction = 'add' | 'replace' as const;
```

Reply:
Ah yes, it's only one specific value: `add`. I've tried this here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66596/commits/6ae1b5c2abb276372ac8e30d972d4ce7374c867f. Does this look OK? Typescript didn't seem to like `as const`, but it otherwise seems to work.

Reply:
not sure what I was thinking there. I guess I was thinking of `const enum` which is different. this looks great.

Reply:
@dmsnell, you were prob thinking of this: https://www.typescriptlang.org/play?#code/MYewdgzgLgBG4GFzQPIAcoEtkwLwwG0ByAQwBMyiAaGIgJwFM0AbE4BogXQG4AoUSLAGoM2SHkKkK1WoxZsOnGCQgxhUPrygBPNAxgA5RMijos4ACq79+HXpAAzOMcFmxEAmACuAWwBGDHQ8vAD0ITARAHrRMbFxMTAAFiAAboEwPvoAtFkwdvqYqgAG0HSYYADmRVrWMEiuopa1ttaOaiZuyJ6+AUF8YREw8cPRSanpmTA5ebWFMEVSlDAAPrJMrOxERUA

`as const` also works for objects in a similar way.